### DISCUSSION TITLE
Possible Biases When Applying the Rodgers & Rowland Model

### DISCUSSION THREAD

**Discussion by wangwei1619** [Open-Systems-Pharmacology/Forum#1683] [Questions & Problems]
Hi, all 

In MOBI, I find that the mass transfer from the interstitium to intracellular space using Rodgers & Rowland Model is defined as **K_water_int * PA_int_cell * C_int - K_water_cell * PA_cell_int * C_cell**, wherein the **K_water_cell** is actually the reciprocal of the tissue to plasma free drug partition coefficient (Kpu) given by  Rodgers & Rowland. 

In the definition of Kpu, the organ is taken as a whole, and therefore the distribution into both extracellular and intracellular components is considered and lumped into the Kpu. 

However, in PK-Sim, where the extracellular and intracellular spaces are separated, the intracellular concentration **C_cell** is modified by the partition coefficient **K_water_cell** which still includes the distribution into the extracellular component. Would this lead to potential biases in simulation?

**Comment by Yuri05** [comment]
We are aware of this issue and are working on an improvement.
The change may affect the simulation results depending on the properties of the used compound etc.
